plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMapping.java

```diff
@@ -39,11 +41,22 @@
 {
     private BigQuerySqlExecutor bigQuerySqlExecutor;
 
+    @BeforeClass
+    public void initBigQueryExecutor()
+    {
+        this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
+    }
+
+    @BeforeTest(alwaysRun = true)
```

Comment:
This will run the method before every test method. I think you meant `@BeforeClass`?